content/millennium/r4/diagnostic/questionnaire-response.md

```diff
@@ -11,11 +11,25 @@ title: QuestionnaireResponse | R4 API
 
 ## Overview
 
-The QuestionnaireResponse resource is a collection of answers to a given questionnaire typically used to request patient healthcare information. An example of a questionnaire is a form used to collect a patient's social history. Currently, only social history is supported.
+The QuestionnaireResponse resource is a collection of answers to a given questionnaire typically used to request patient healthcare information. Only answer value types of String, Quantity, and Coding are supported. An example of a questionnaire is a form used to collect a patient's social history. Currently, only social history, which is a type of Observation, is supported. To find the corresponding social history questionnaire, first search QuestionnaireResponse by patient. As long as the patient exists, a QuestionnaireReponse will be returned even when the data does not exist. Take the 'questionnaire' field from the QuestionnaireResponse and use that to retrieve the original Questionnaire.
 
 The following fields are returned if valued:
 
 * [Id](https://hl7.org/fhir/R4/resource-definitions.html#Resource.id){:target="_blank"}
+* [Questionnaire](http://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.questionnaire){:target="_blank"}
+* [Status](http://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.status){:target="_blank"}
+* [Subject](http://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.subject){:target="_blank"}
+* [Encounter](http://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.encounter){:target="_blank"}
+* [Authored](http://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.authored){:target="_blank"}
+* [Item](http://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.item){:target="_blank"}
+    * [Link id](http://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.item.linkId){:target="_blank"}
+    * [Text](http://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.item.text){:target="_blank"}
+    * [Item](http://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.item.item){:target="_blank"}
+        * [Link id](http://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.item.linkId){:target="_blank"}
+        * [Text](http://hl7.org/fhir/r4/questionnaireresponse-definitions.html#QuestionnaireResponse.item.text){:target="_blank"}
+        * [valueString Answers](http://hl7.org/fhir/r4/datatypes.html#string){:target="_blank"}
+        * [valueCoding Answers](http://hl7.org/fhir/r4/datatypes.html#Coding){:target="_blank"}
+        * [valueQuantity Answers](http://hl7.org/fhir/r4/datatypes.html#Quantity){:target="_blank"}
 
```

Comment:
We will have a terminology binding just like we did for questionnaire: https://github.com/cerner/fhir.cerner.com/blame/main/content/millennium/r4/definitional-artifacts/questionnaire.md#L36 for the nomenclatures

Reply:
We also need an extensions section for the choice-answer and note extensions

Reply:
aa3f60c679b1fce87f0298d0e7d999f76946d230

Reply:
just the note extension, note-allowed is only on questionnaire

Reply:
1c8154bbdb58cf93f0a5a23ed377a74b4a4583b2

Reply:
'Freetext details about a given element.' isnt right... ummm the StructureDef is 'Additional details related to an element within the resource along with author and date/time information.'
So to make that specific to QR, 'Additional details related to the QuestionnaireResponse item with author and date/time information.' ?

Reply:
My bad, I took the shortened definition from the structure definition. I think what you have is pretty good: 9734a8bceeb9af963a16add1c38ca64a8857be18